.github/workflows/codespell.yml

```
@@ -0,0 +1,25 @@
+name: Codespell
+
+on: ['pull_request']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v26.1
+        with:
+          # Ignore all other languages except English
+          files_ignore: pages.*/*/* 
+        
+      - uses: codespell-project/actions-codespell@master
+        with:
+          ignore_words_file: .github/codespell-ignorelist
```

The ignore files is always a list.
```suggestion
          ignore_words_file: .github/codespell-ignore
```
Also could be `.codespellignore` following the `.gitignore`, `.dockerignore`, `.npmignore` formats
```suggestion
          ignore_words_file: .github/.codespellignore
```